apps/projects/contracts/Projects.sol

```diff
@@ -359,8 +363,8 @@ contract Projects is AragonApp, DepositableStorage {
 // Repository functions
 ///////////////////////
     /**
-     * @notice Add repository to the Projects app
-     * @param _repoId id of the repo to be added
+     * @notice Add repository `_repoId`
```

Comment:
I don't think these repo IDs are human-readable, and I doubt it makes sense to put them in the radspec. I think they're just going to look sort of like ugly ethereum-address-style hashes.

Reply:
Yeah but there is no other context for the voting app "Add repository" doesn't convey anything. I'm not sure how we can properly solve this, but it's a UX concern if we want people to be able to vote to add or remove a repo. I was hoping maybe we can provide with some kind of tool to look up the repo name via Github app.

Reply:
I've decided to add a metadata field here, which we can presumable pass the repo name into from the github repo object we have in the frontend